extensions/amp-subscriptions/0.1/test/test-local-subscriptions.js

```diff
@@ -106,12 +108,24 @@ describes.fakeWin('LocalSubscriptionsPlatform', {amp: true}, env => {
         'buildUrl').callsFake(() => Promise.resolve(builtUrl));
     const fetchStub = sandbox.stub(localSubscriptionPlatform.xhr_,'fetchJson')
         .callsFake(() => Promise.resolve({json: () => Promise.resolve(json)}));
+    sandbox.stub(localSubscriptionPlatform.serviceAdapter_,
+        'getEncryptedDocumentKey');
     return localSubscriptionPlatform.getEntitlements().then(() => {
       expect(urlBuildingStub).to.be.calledWith(configAuthUrl, false);
       expect(fetchStub).to.be.calledWith(builtUrl, {credentials: 'include'});
     });
   });
 
+  it('should call getEncryptedDocumentKey with google.com', () => {
+    const getEncryptedDocumentKeyStub = sandbox.stub(
+        localSubscriptionPlatform.serviceAdapter_,
+        'getEncryptedDocumentKey'
+    );
+    return localSubscriptionPlatform.getEntitlements().then(() => {
+      expect(getEncryptedDocumentKeyStub).to.be.calledWith('google.com');
```

Comment:
I might be missing something, but why should the `LocalPlatform` be asking for `google.com` key? Also made a corresponding comment on the `LocalPlatform` itself.

Reply:
fixed